packages/create-sitecore-jss/src/templates/angular/scripts/generate-config.ts

```diff
@@ -67,9 +67,12 @@ export function writeConfig(config: JssConfig, outputPath?: string) {
 
   // Set base configuration values, allowing override with environment variables
   Object.keys(config).forEach((prop) => {
-    configText += `config.${prop} = process.env.${constantCase(prop)} || "${config[prop]
-      ?.toString()
-      .trim()}";\n`;
+    let value = config[prop]?.toString().trim();
+    const usesTemplateLiteral = /\$\{.*?\}/.test(config[prop].toString());
+
+    value = usesTemplateLiteral ? `\`${value}\`` : `"${value}"`;
+
+    configText += `config.${prop} = process.env.${constantCase(prop)} || ${value};\n`;
```

Comment:
Just a reminder to re-evaluate the need for this (template literal support) based on decision for `graphQLEndpoint` config generation.

Reply:
yes! reverted